oauth2.go

```diff
@@ -55,6 +55,15 @@ type Application struct {
 	Owner               *User     `json:"owner"`
 	Bot                 *User     `json:"bot"`
 	Team                *Team     `json:"team"`
+	VerifyKey           string    `json:"verify_key"`
+	RPCOrigins          *[]string `json:"rpc_origins,omitempty"`
```

Comment:
Is having a pointer to the slice really necessary here? The field is not nullable, and I don't think that we really need to differentiate between present and not present—I don't think that the field would be sent if the array were empty.

Reply:
Fixed!